subcommittee/coding-guidelines/meetings/19-November-2024/minutes.md

```diff
@@ -0,0 +1,146 @@
+# Coding Guidelines Subcommittee Meeting on 2024-11-19
+
+## Agenda
+
+0. Meeting recordings / transcriptions - Live discussion
+1. Acceptance of Previous Meeting Minutes
+2. Shift meeting time 2 hours earlier to accommodate Asia participants (Pete LeVasseur)
+3. Contributor Expectations - Approval (Pete LeVasseur)
+4. Safe Usage of Unsafe Guidelines - Check-in (Pete LeVasseur)
+    a. Learn unsafe Rust - Offline Review & Q&A
+    b. Examples of unsafe patterns - Chapter writeup
+5. MISRA + Rust (Alex Celeste)
+6. Round table
+
+Supplemental material to the agenda can be found on the [GitHub repo](https://github.com/rustfoundation/safety-critical-rust-consortium/tree/main/subcommittee/coding-guidelines).
+
+## Check-in area
+
+**Please add your name, and an emoji that describes your day.**
+
```

Comment:
I really need to remember to add my name before the meeting gets rolling next time 😆